runtime/bindings/python/src/pyopenvino/graph/ops/parameter.cpp

```diff
@@ -35,4 +35,22 @@ void regclass_graph_op_Parameter(py::module m) {
     parameter.def("get_partial_shape",
                   (ov::PartialShape & (ov::op::v0::Parameter::*)()) & ov::op::v0::Parameter::get_partial_shape);
     parameter.def("set_partial_shape", &ov::op::v0::Parameter::set_partial_shape);
+
+    parameter.def("get_element_type", &ov::op::v0::Parameter::get_element_type);
+
+    parameter.def("set_element_type", &ov::op::v0::Parameter::set_element_type);
+
+    parameter.def("get_layout", &ov::op::v0::Parameter::get_layout);
+
+    parameter.def("set_layout", &ov::op::v0::Parameter::set_layout);
+
+    parameter.def_property("partial_shape",
```

Comment:
@jiwaszki Do we need to have both approaches with getters/setters and properties?

Reply:
The pythonic way is use of properties but we kept getters/setters to be aligned with cpp